### DISCUSSION TITLE
Stars view only updating after manual refresh

### DISCUSSION THREAD

**Discussion by andreaswilli** [community/community#75625] [Mobile] [answered]
I opened the stars view on one of my repos. Since I last opened it, one new star was added. However, the new entry only showed up after a manual refresh. Is this a bug?

https://github.com/community/community/assets/17298270/7048563d-c338-4b8c-9411-e98f067bafc5

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.138.0 (115117336)
Device: iPhone 11 (iOS 17.0)
</details>

**Comment by jamirou** [comment]
I have had a similar problem in the new iOS update. I haven't paid much attention to it but I suppose it's one of those bugs that appear in some "incompatibilities".

**Comment by andreaswilli** [comment, marked as answer]
Seems to be fixed now.
